app/models/datawarehouse_node.rb

```diff
@@ -0,0 +1,9 @@
+class DatawarehouseNode < ApplicationRecord
```

Comment:
@enoodle I guess you mean provider specific. So we agree on not having `Elastic` here. And you are right, with this not being provider specific, it's rather manager type specific (all these terms)...

It's just there are so many `container_` and `cloud_` models already at the root level.
I would like to see a `node` -> `dwh_node` -> `hawkular_node` inheritance.
If we can't re-use an existing or new shared node model. I'm good with `dwh_node`

Maybe you explore to move it to a `Datawarehouse` namespace?

Like `class Datawarehouse::Node` ?